plugin/secrets_service_account_key.go

```diff
@@ -99,7 +99,10 @@ func (b *backend) secretKeyRenew(ctx context.Context, req *logical.Request, d *f
 		return nil, err
 	}
 	if cfg == nil {
-		cfg = &config{}
+		cfg = &config{
```

Comment:
This change is not required.  You can just pass the zero values back and they will get set appropriately.